__tests__/api.test.js

```diff
@@ -0,0 +1,33 @@
+const app = require("../app")
+const request = require('supertest')
+const db = require('../db/connection')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data/index');
+
+beforeEach(() => {
+    return seed(testData)
+});
+afterAll(() => {
+    return db.end()
+});
+
+describe('GET /api/topics', () => {
+    test('200: responds with an array of topic objects', () => {
+        return request(app)
+            .get("/api/topics")
+            .expect(200)
+            .then(({ body }) => {
+                body.data.forEach((topic) => {
```

Comment:
As we're using a forEach here, we should check the length of the array beforehand 👍